src/figuresClasses.ts

```diff
@@ -1,19 +1,112 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea: ()=> number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
+
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(
+    color: string, a: number, b: number, c: number,
+  ) {
```

Comment:
you can access Figure types with this syntax: `Figure['typeName']` - you won't have to retype everything again